src/service_name.json

```diff
@@ -716,47 +696,42 @@
   },
   {
     "Command": "az billing-benefits",
-    "AzureServiceName": "Azure Billing Benefits",
-    "URL": "https://learn.microsoft.com/en-us/azure/cost-management-billing/savings-plan/"
+    "AzureServiceName": "Cost Management",
+    "URL": "https://learn.microsoft.com/azure/cost-management-billing/savings-plan"
   },
   {
     "Command": "az mobile-network",
-    "AzureServiceName": "Mobile Network",
-    "URL": "https://learn.microsoft.com/en-us/azure/private-5g-core/"
+    "AzureServiceName": "Mobile Networking",
+    "URL": "https://learn.microsoft.com/azure/private-5g-core"
   },
   {
     "Command": "az automanage",
-    "AzureServiceName": "Azure Automanage",
-    "URL": "https://learn.microsoft.com/en-us/azure/automanage/"
-  },
-  {
-    "Command": "az voice-service",
-    "AzureServiceName": "Azure VoiceServices",
-    "URL": ""
+    "AzureServiceName": "Virtual Machines",
+    "URL": "https://learn.microsoft.com/azure/automanage"
   },
   {
     "Command": "az storage-mover",
-    "AzureServiceName": "Storage Mover",
-    "URL": "https://learn.microsoft.com/en-us/azure/storage-mover/"
+    "AzureServiceName": "Storage",
+    "URL": "https://learn.microsoft.com/azure/storage-mover"
   },
   {
-    "Command": "az confidentialledger managedccfs",
-    "AzureServiceName": "Azure Managed Confidential Consortium Framework",
-    "URL": "https://docs.microsoft.com/en-us/azure/confidential-ledger/"
+    "Command": "az confidentialledger",
+    "AzureServiceName": "Confidential Consortium Framework (CCF)",
+    "URL": "https://learn.microsoft.com/azure/confidential-ledger"
   },
   {
     "Command": "az workloads",
-    "AzureServiceName": "Azure Workloads",
-    "URL": "https://learn.microsoft.com/en-us/azure/sap/workloads/get-started"
+    "AzureServiceName": "SAP on Azure",
+    "URL": "https://learn.microsoft.com/azure/sap/workloads/get-started"
   },
   {
     "Command": "az self-help",
-    "AzureServiceName": "Azure Help",
-    "URL": "https://learn.microsoft.com/en-us/rest/api/help/"
+    "AzureServiceName": "Azure CLI",
+    "URL": "https://learn.microsoft.com/rest/api/help"
```

Comment:
@dbradish-microsoft / @jsntcy this related to an Azure API, we should put it under "Self Help", not "Azure CLI", this could be misleading.